Assets/MixedRealityToolkit/Definitions/MixedRealitySpatialObserverConfiguration.cs

```diff
@@ -30,11 +30,25 @@ public struct MixedRealitySpatialObserverConfiguration : IMixedRealityServiceCon
         public uint Priority => priority;
 
         [SerializeField]
-        [EnumFlags]
-        private SupportedPlatforms runtimePlatform;
+        [Implements(typeof(IPlatformSupport), TypeGrouping.ByNamespaceFlat)]
+        private SystemType[] runtimePlatform;
 
         /// <inheritdoc />
-        public SupportedPlatforms RuntimePlatform => runtimePlatform;
+        private IPlatformSupport[] _runtimePlatform;
+
+        /// <inheritdoc />
+        public IPlatformSupport[] RuntimePlatform
+        {
+            get
+            {
+                if (_runtimePlatform == null)
```

Comment:
based on the description, null is the same as everything. null should be the equivalent of "none"

Reply:
you sure? none wasn't available before. It's the only option to even have an "all platforms" choice, because otherwise classes using the new system would need to include all of them manually.
Alternatively, of course there could be an "All Platforms" class that just returns true no matter what. But what's the use of none?